src/server/index.jsx

```diff
@@ -123,6 +149,21 @@ server
       }
     });
   })
+  .get(homePageRegexPath, async ({ url, headers }, res) => {
```

Comment:
Just being nosey, but we should aim to use `/*` as our route here and let react router handle it all :)

Reply:
Agree, although that is not in place at the moment right?

Reply:
Nope, we had to make it used a regex as before when we used @jtart's React universal app, the error thrown for missing a route https://github.com/bbc/simorgh/blob/latest/src/app/routes/loadInitialData/index.js#L11 couldn't be distinguished between a route error and a actual error

We should handle it differently so they can be distinguished